.github/workflows/documentation.yml

```diff
@@ -0,0 +1,41 @@
+name: Documentation
+on:
+  push:
+    branches:
+    - main
```

Comment:
If you run on pushes to main the documentation won't match the released version.

```suggestion
    tags:
      - v*
```

Reply:
Changed the trigger for tags